tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -385,11 +384,7 @@ def diff_strings(a: str, b: str, *, use_loguru_colors: bool = False) -> str:
 
 
 def genai_gen_answer(model, tokenizer, question, max_new_tokens, skip_question):
-    config = openvino_genai.GenerationConfig()
-    config.max_new_tokens = max_new_tokens
-    config.do_sample = False
-    out = model.generate(question, config)
-    return out
+    return model.generate(question, do_sample=False, max_new_tokens=max_new_tokens)
```

Comment:
Can we do the same thing in the samples we have?

Reply:
The samples intentionally avoid this to get more deterministic results depending only on GenAi defaults.

Reply:
Thanks, it is fair enough. But how do we educate the user to get the same generation results when they transition from HF API to OV GenAI which is likely to happen?

Reply:
Fair point :) @ilya-lavrenov, can I update the samples to preserve models' defaults and  freeze `max_new_tokens` and `do_sample` only?

Reply:
> Fair point :) @ilya-lavrenov, can I update the samples to preserve models' defaults and freeze max_new_tokens and do_sample only?

Agree

Reply:
Sure. I just can't pass tests here